Gemfile

```
@@ -15,6 +15,7 @@ gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw, :jruby] # Windows doe
 # gem 'redis' # Use Redis adapter to run Action Cable in production
 # gem 'kredis' # Use Kredis to get higher-level data types in Redis
 # gem 'bcrypt' # Use Active Model has_secure_password
+gem 'devise'
```

let's be consistent with other gem
```suggestion
gem 'devise' # Flexible authentication solution for Rails with Warden
```